src/index.js

```diff
@@ -12,6 +12,7 @@ export * from './get-node-text'
 export * from './events'
 export * from './get-queries-for-element'
 export * from './query-helpers'
+export * from './role-helpers'
```

Comment:
I love this PR! I learned a ton by reading your code and comments. 

Now that you've discussed public API, I'm still not sold on the idea of the current naming. I feel that, as exposed in https://github.com/testing-library/dom-testing-library/issues/282#issuecomment-502298508, `getRoles` could easily be confused with one of the `getBy*` queries from DTL. What do you folks think about that?

Reply:
I don't think all that many people do this:

```javascript
import {getByRole} from '@testing-library/dom'
```

I'm pretty sure most people do this:

```javascript
import {render} from '@testing-library/some-implementation'

const {getByRole} = render(myThing)
```

Or this:

```javascript
import {getQueriesForElement} from '@testing-library/dom'

const {getByRole} = getQueriesForElement(someContainer)
```

So I'm not very concerned about this.

Reply:
Good catch on the exports, I had completely forgotten to revisit those.  

I've removed the elementRoleList export, that is certainly not needed anymore, now that getImplicitAriaRoles is moved from src/queries/role to src/role-helpers.

We do need to export getImplicitAriaRoles  from role-helpers because src/queries/role uses it.  That said, I don't think it really needs to be part of the public API.  

Should we just export getRoles and logRoles in index?  Will commit that change after hearing back.

Reply:
Yeah, I think that's right 👍

Reply:
[Done b9a16e1](https://github.com/testing-library/dom-testing-library/pull/285/commits/b9a16e1f27d4f57e53d16b992563f4b1d4097303)

I'll get started on documentation tonight or tomorrow as well, now that things are starting to come together.